app/src/main/java/org/mozilla/fenix/settings/SupportUtils.kt

```diff
@@ -0,0 +1,48 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.settings
+
+import android.content.Context
+import android.content.pm.PackageManager
+import java.io.UnsupportedEncodingException
+import java.net.URLEncoder
+import java.util.Locale
+
+object SupportUtils {
+    const val FEEDBACK_URL = "https://input.mozilla.org"
+    const val RATE_APP_URL = "market://details?id=org.mozilla.fenix"
```

Comment:
This seems really minor, but it would be better to fill in the app id from BuildConfig.APPLICATION_ID, just in case. This would help in the case where another company ripped off our open source and published it in the store. Not like that's ever happened. ;)